src/main/java/com/example/demo/repository/CityRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.example.demo.repository;
+
+import com.example.demo.dto.CityDto;
+
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+
+public interface CityRepository extends JpaRepository<CityDto, Integer> {
```

Comment:
```suggestion
public interface CityRepository extends JpaRepository<City, Integer> {
```
El repo deberia ser de City